tlvc/arch/i86/drivers/block/ll_rw_blk.c

```diff
@@ -261,12 +247,13 @@ static void make_request(unsigned short major, int rw, struct buffer_head *bh)
     }
 
     /* fill up the request-info, and add it to the queue */
+    clr_irq();
```

Comment:
Conversely to the comment above, disabling interrupts isn't required here because `get_request`, which is always called with interrupts disabled, sets `req->rq_status = RQ_ACTIVE` after selecting an RQ_INACTIVE request. This guarantees the request won't be found in any driver queue and thus not referenced by an interrupt routine.

Reply:
Thanks, good point!